blivet/devicelibs/mdraid.py

```diff
@@ -343,6 +343,9 @@ def mddetail(device):
 
     info = {}
     for (name, colon, value) in (line.strip().partition(":") for line in lines):
+        if name and name[0] not in string.letters:
+            continue
```

Comment:
In doing the legwork to reproduce and thus obtain the sample output, it occurred to me that I could make a much "simpler" _change_ to the parsing code instead of adding this hack. While this sounds nice, it is a change on `rhel7-branch`, so I'm not sure the risk is worth taking for the sake of nicer code.

Here's what it looks like (diff from this pull request branch):

```patch
diff --git a/blivet/devicelibs/mdraid.py b/blivet/devicelibs/mdraid.py
index 33c1ec0..4906719 100644
--- a/blivet/devicelibs/mdraid.py
+++ b/blivet/devicelibs/mdraid.py
@@ -342,10 +342,7 @@ def mddetail(device):
         raise MDRaidError("mddetail failed for %s: %s" % (device, e))
 
     info = {}
-    for (name, colon, value) in (line.strip().partition(":") for line in lines):
-        if name and name[0] not in string.letters:
-            continue
-
+    for (name, colon, value) in (line.strip().partition(" : ") for line in lines):
         value = value.strip()
         name = name.strip().upper()
         if colon and value and name:
```

Opinions?